packages/bidi.lua

```diff
@@ -128,7 +128,7 @@ local reorder = function (n, self)
           reverse_each_node(nl[i].postbreak[j].nodes)
         end
       end
-    elseif nl[i]:isAlternative() and levels[i].level %2 ~= base_level then
+    elseif nl[i]:is_alternative and levels[i].level %2 ~= base_level then
```

Comment:
The `:isAlternative()` method should still work, of you need a faster test `.is_alternative` should be a preset boolean.